src/models/Scholarships.test.ts

```diff
@@ -270,19 +270,6 @@ test('scholarships.list - filters by minAmount', async () => {
   );
 });
 
-test('scholarships.list - filters by maxAmount', async () => {
```

Comment:
I don't think I would worry about removing it from `src/lib` and `src/models` yet. It might prove useful later if we do identify a need for it. We can just remove it from the UI for now.